app/frontend/src/features/messages/messagelist/InfiniteMessageLoader.tsx

```diff
@@ -91,15 +93,25 @@ export default function InfiniteMessageLoader({
     return () => window.removeEventListener("resize", updateMessagePosition);
   }, []);
 
-  //** Scroll to the bottom on new message  **//
-  const [savedMessageId, setSavedMessageId] = useState(latestMessageId);
-  useEffect(() => {
+  //** Scroll to the bottom after sending own new message  **//
+  const [savedMessageId, setSavedMessageId] = useState(
+    latestMessage?.messageId
+  );
```

Comment:
Sorry for not noticing this earlier. It would be more correct for this to use `useRef` and therefore not be in the layout effect dependency array. When you set savedMessageId, you don't actually want to trigger a re-render or run the layout effect again, you're just saving it for later comparison, and it's `latestMessage` that triggers the layout effect.

As is, the layout effect gets triggered again after you `setSavedMessageId` (but the second time obviously doesn't do anything.)

Reply:
I think i've done this lol